### PR TITLE
W3C validation passes on all pages

### PR DESCRIPTION
**Pull request by gggrant**
### Notes:
- https://validator.w3.org/
- Used this extension for testing: https://www.validity.org.uk/

### CODE REVIEW

File: resources/views/common/modals/create_reference.html.twig [hunk 1]
```diff
@@ -193,4 +194,4 @@
 
     </div>
 
-</div>
+<!--</div>-->
```
Comment (Khristophor) [inline resources/views/common/modals/create_reference.html.twig]: Curious why this closing tag is commented?

Comment (gggrant) [reply resources/views/common/modals/create_reference.html.twig]: Removed it, was a stray end tag on reference pages.

Comment (Khristophor) [reply resources/views/common/modals/create_reference.html.twig]: Are we sure this particular close tag is the culprit? This is a pretty small file with paired open/close tags... I have a suspicion the larger more complicated template that includes this one is the problem.

Comment (gggrant) [reply resources/views/common/modals/create_reference.html.twig]: I've restored it. Can't reproduce the stray tag error now.

**Pull request merged**